addons/account/tools/__init__.py

```diff
@@ -1,28 +1,7 @@
-from itertools import zip_longest
 import requests
 from urllib3.util.ssl_ import create_urllib3_context
 
-
-def calc_check_digits(number: str) -> str:
```

Comment:
Then what about use of these methods in existing code ?...
It's not stable at all...
https://www.odoo.com/odoo/project/49/tasks/4636613